app/controllers/people_controller.rb

```diff
@@ -37,7 +37,13 @@ def update
   def show
     @person = Person.find(params[:id])
   end
-
+  
+  def search
+    @person = Person.find(params[:id])
+    @link = "https://www.google.pl/search?q=(person_params
+    )+(person_params)"
```

Comment:
To powinno być w jednej linijce, a imię i nazwisko trzeba pobrać ze zmiennej @person (`@person[:name]`, `@person[:surname]`). Trzeba też będzie użyć wywołania `redirect_to`.